README.md

```diff
@@ -0,0 +1,106 @@
+    - *Bill*
+      - Class representing a bill object. Allows creation of bills with their respective names, amount due, due dates, and funding status.
+    - *BillWidget*
+      - Class representing the widget the user will use to enter the bills they wish to keep track of. This is a widget they will utilize upon first launch, as well as if they choose to add additional bills on subsequent runs of the application.
+    - *MainWindow*
+      - Class containing the primary user interface (UI) of the application. The financial dashboard is displayed here and the majority of the processing of user data occurs here.
+    - *main*
+      - Main file to execute the program.
+- *.gitignore*
+  - Used for indicating which folders and files should not be tracked by git.
+  - *PersonalFinanceTool.pro*
+    - Qt project file which contains configuration flags, the included source and header files, and more.
+  - *README.md*
+    - This file which serves as supplemental material to help a user better understand how to obtain, install, and utilize the PFT.
+
+## Execution
+
+Assuming the prerequisites from above have been met and the source code has been cloned, the user should perform the following steps to run the PFT:
+
+1. Change directory into the *PersonalFinanceTool* folder.
+2. `qmake`
+3. `make clean`
+4. `make`
+5. `sudo make install`
+6. The user should see they now have a *bin* folder within the *PersonalFinanceTool* folder. Run the executable file inside that folder.
+
+If all was successful, the user should receive a pop-up welcoming them to the PFT.
+
+## Usage
+
+The following sections will detail various functionalities of the PFT.
+
+### Entering Total Amount Available
+
+Upon the initial run of the PFT, the user will receive a welcome pop-up, followed by a prompt to enter the total amount of money they have available in United States Dollars (USD).
+
+![Entering Total Amount Available](/images/total_amount_available_request.png)
+
+### Bill Entry
+
+Following the total amount available request, the user will be prompted to enter the bills they wish to keep track of via the bill widget. They'll be asked for the name of the bill, the amount due in USD, and the due date. When entering the due date, they can either manually type it in in *mm/dd/yyyy* format, or utilize the calendar view to select the date instead.
+
+![Bill Entry](/images/bill_entry.png)
+
+If the user does not enter a name for the bill and/or the amount due, they will receive an error message indicating there's missing bill information.
+
+![Missing Bill Information](/images/missing_bill_information.png)
+
+While entering bills, the user can click the *Close* button to terminate the application, the *Enter Another Bill* button to save the currently entered bill and relaunch the Bill Entry widget, and the *Done* button to save the currently entered bill and display the financial dashboard.
```

Comment:
Change *"and relaunch the Bill Entry widget"* to *"and reopen the bill widget for entry of another bill"*

Reply:
Corrected.